erpnext/education/doctype/quiz_question/quiz_question.json

```diff
@@ -20,7 +20,7 @@
   {
    "fetch_from": "question_link.question",
    "fieldname": "question",
-   "fieldtype": "Data",
+   "fieldtype": "Text Editor",
```

Comment:
This is the child table in Quiz doctype which fetches the question from **Question** doctype. So you will have to change the fieldtype to Text Editor over there too.

![question](https://user-images.githubusercontent.com/24353136/94127481-9f464500-fe76-11ea-816e-a071852fc717.png)

Reply:
What you have mentioned does make sense. It is supposed to be like that. however I did not notice any change in behavior in that